### PR TITLE
Fix race conditions with webpack_dev_server

### PR DESCRIPTION
**Pull request by mrmeku**
- Export symbols from index.bzl
- Include webpack_dev_server in CI
- Release version 0.2.0

### CODE REVIEW

File: e2e/package.json [hunk 1]
```diff
@@ -2,13 +2,12 @@
   "name": "e2e",
   "version": "1.0.0",
   "scripts": {
-    "test": "bazel test //..."
+    "test": "bazel test //... && bazel build //..."
```
Comment (alexeagle) [inline e2e/package.json:5]: did you actually need this? `test` should include building everything buildable (unless you use `--build_tests_only`)

File: e2e/WORKSPACE [hunk 2]
```diff
@@ -15,8 +15,8 @@ http_archive(
 
 http_archive(
```
Comment (alexeagle) [inline e2e/WORKSPACE:16]: we should probably call the `repositories.bzl` helper from rules_webpack instead of an extra fetch of build_bazel_rules_nodejs

**Review by alexeagle — APPROVED** [review]

File: e2e/package.json (hunk 1 above)
Comment (mrmeku) [reply e2e/package.json:5]: No, didn't end up needing it

File: e2e/WORKSPACE (hunk 2 above)
Comment (mrmeku) [reply e2e/WORKSPACE:16]: I don't think we were fetching twice since, but I agree that we should use `repositories.bzl`.
Done

**Pull request merged**